theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -314,11 +317,12 @@ def fn(x):
             # Test the GPU grad + GPU implementation
             def fn(x):
                 dnn_op = cuda.dnn.dnn_pool(
-                    x, ws=(ws, ws),
-                    stride=(stride, stride),
-                    pad=pad,
+                    x, ws=ws,
+                    stride=stride,
+                    pad=pad_,
```

Comment:
Ohh you mean don't change this test, and make a separate test to verify if it works with tensor variables?